packages/client/index.d.ts

```diff
@@ -0,0 +1,56 @@
+
+import { FastifyPluginAsync } from 'fastify'
+
+interface Headers {
+  [key: string]: string
+}
+
+interface PlatformaticClientPluginOptions {
+  getHeaders: () => Headers;
+  serviceId: string;
+  url: string;
+  type: 'openapi' | 'graphql';
+  name: string;
+}
+
+interface BuildOpenAPIClientOptions {
+  url: string;
+  path: string;
+  headers?: Headers
+}
+
+interface AbstractLogger {
+  fatal: () => void;
+  error: () => void;
+  warn: () => void;
+  info: () => void;
+  debug: () => void;
+  trace: () => void;
+}
+
+interface BuildGraphQLClientOptions {
+  url: string;
+  headers?: Headers;
+}
+
+interface Parameter {
+  in: string;
+  name: string;
+}
+
+interface MethodMetaInterface {
+  operationId?: string,
+  parameters: Parameter[]
+}
+
+interface BuildGraphQLClientOutput {
+  graphql: () => Promise<unkown>;
+  [Symbol('headers')]: Headers; 
+}
+
+export function generateOperationId(path: string, method: string, methodMeta: MethodMetaInterface): string
+export async function buildOpenAPIClient(options: BuildOpenAPIClientOptions, openTelemetry: any): Promise<BuildGraphQLClientOutput>
```

Comment:
```suggestion
export async function buildOpenAPIClient(options: BuildOpenAPIClientOptions, openTelemetry: any): Promise<BuildOpenAPIClientOutput>
```

Note that `BuildOpenAPIClientOutput` does not exit.

Reply:
My bad, it was a typo